.gitignore

```diff
@@ -2,3 +2,4 @@ target/
 node_modules/
 WEB-INF/
 trending_data/src/main/resources/config.properties
+key.txt
```

Comment:
I'm not 100% sure about this, but if the read_maps_key.js is reading the file that is 'key.txt', does that mean that key.txt is in the same folder? If that's the case, should this line be trending_data/src/main/webapp/key.txt?

Reply:
Yes it is in the webapp folder, very weird that it worked in the first place. I'll change it to include the whole path